packages/babel-plugin-transform-es2015-typeof-symbol/src/index.js

```diff
@@ -3,6 +3,14 @@ export default function ({ types: t }) {
 
   return {
     visitor: {
+      Scope({ scope }) {
+        if (!scope.getBinding("Symbol")) {
+          return;
+        }
+
+        scope.rename("Symbol");
```

Comment:
Should this just be a feature of Babel Core? This fixes this specific case, but this'll happen anywhere any global is shadowed. For instance

```
var Symbol = undefined;
var a;
a = [1, 2, 3];
for (var i of a){
    console.log(i);
}
```

would also throw for this reason.

I assume this same issue would happen with our use of `Object` and `Promise`, not sure if there are other globals we use in the helpers.